tensorflow_datasets/image/imagenette.py

```diff
@@ -0,0 +1,131 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Imagenette: a subset of 10 easily classified classes from Imagenet (tench,
+English springer, cassette player, chain saw, church, French horn, garbage
+truck, gas pump, golf ball, parachute)"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+
+# TODO(imagenette): BibTeX citation
+_CITATION = """
+"""
+
+_DESCRIPTION = """\
+Imagenette is a subset of 10 easily classified classes from the Imagenet
+dataset. It was originally prepared by Jeremy Howard of FastAI. The objective
+behind putting together a small version of the Imagenet dataset was mainly
+because running new ideas/algorithms/experiments on the whole Imagenet take a
+lot of time.
+
+This version of the dataset allows researchers/practitioners to quickly try out
+ideas and share with others. The dataset comes in three variants:
+
+  * Full size
+  * 320 px
+  * 160 px
+This dataset consists of the Imagenette dataset {variant} variant.
+"""
+
+_LABELS_FNAME = 'image/imagenette_labels.txt'
+_URL_PREFIX = 'https://s3.amazonaws.com/fast-ai-imageclas'
+_VARIANTS = ["imagenette-160.tgz", "imagenette-320.tgz", "imagenette.tgz"]
+
+
+class ImagenetteConfig(tfds.core.BuilderConfig):
+  """BuilderConfig for Imagenette."""
+
+  def __init__(self, **kwargs):
+    super(ImagenetteConfig, self).__init__(
+        version=tfds.core.Version('0.1.0'), **kwargs)
+
+
+def _make_builder_configs():
+  configs = []
+  for variant in _VARIANTS:
```

Comment:
BTW -  having the config name "imagenette-160.tgz" doesn't look nice 
(as people will have to say:

tfds.load("imagenette/imagenette-160.tar")

Could you have the config named 160px instead ?

Reply:
So use 160px instead of imagenette-160.tgz as the `name` parameter in `ImagenetteConfig`?